compiler/src/dotty/tools/dotc/tastyreflect/TastyImpl.scala

```diff
@@ -2,4 +2,4 @@ package dotty.tools.dotc.tastyreflect
 
 import dotty.tools.dotc.core._
 
```

Comment:
Format the following line into several lines?

Reply:
Done